### PR TITLE
🌱 reorder OWNERS alphabetically

### PR DESCRIPTION
**Pull request by furkatgofurov7**
**What this PR does / why we need it**:
reorders OWNERS file alphabetically.

### CODE REVIEW

**Comment by furkatgofurov7** [comment]
Unrelated CI problem, re-triggering.

/test-integration

**Review by kashifest — APPROVED** [review]

**Review by maelk — APPROVED** [review]
not really needed, but why not :)

File: OWNERS [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
- - maelk
- - kashifest
-
+ 
 reviewers:
- - Xenwar
- - Jaakko-Os
  - fmuyassarov
  - furkatgofurov7
+ - Jaakko-Os
```
Comment (jan-est) [inline OWNERS:14]: Should we remove Jaakko from here?

Comment (furkatgofurov7) [reply OWNERS:14]: Was not sure, if others also agree, I can remove.

**Review by smoshiur1237 — COMMENTED** [review]
LGTM

**Comment by fmuyassarov** [comment]
/hold
/lgtm
Leaving it to you @furkatgofurov7 to remove the hold once you decide if you want to remove the Jaakko or not.

File: OWNERS (hunk 1 above)
Comment (furkatgofurov7) [reply OWNERS:14]: Since we have him in other repos also, let's keep for now, and remove at once from all repos if everyone agrees.

**Comment by furkatgofurov7** [comment]
> Leaving it to you @furkatgofurov7 to remove the hold once you decide if you want to remove the Jaakko or not.

ref: https://github.com/metal3-io/cluster-api-provider-metal3/pull/173#discussion_r586369123

/unhold

**Pull request merged**